ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -293,10 +291,7 @@ public ThreeDSecureV1UiCustomization getV1UiCustomization() {
         return mV1UiCustomization;
     }
 
-    public ThreeDSecureRequest() {
-        // NOTE: this is a temporary fix for a null-pointer bug introduced by Cardinal 2.2.3-2
```

Comment:
This bug still exists in Cardinal 2.2.5-1, but setting an empty `UiCustomization` here is no longer necessary, since one is instantiated when a `ThreeDSecureV2UiCustomization` is created.